app/presenters/iiif_print/iiif_manifest_presenter_behavior.rb

```diff
@@ -8,7 +8,9 @@ module IiifManifestPresenterBehavior
     attr_accessor :base_url
 
     def manifest_metadata
-      @manifest_metadata ||= IiifPrint.manifest_metadata_from(work: model, presenter: self)
+      # ensure we are using a SolrDocument
+      solr_doc = model.is_a?(SolrDocument) ? model : model.solr_document
```

Comment:
Sure would love `SolrDocument` to respond to `solr_document`; then we wouldn't need the conditional.  But that's a nice to have.

Reply:
We do have a `SolrDocument` decorator, can we leverage that and define `#solr_document` that returns `self`?